src/H5FDhdfs.c

```diff
@@ -1262,8 +1262,8 @@ H5FD__hdfs_close(H5FD_t *_file)
  *     field-by-field.
  *
  * Return:
- *     + Equivalent:      0
- *     + Not Equivalent: -1
+ *      Equivalent:      0
+ *      Not Equivalent: -1
  *
  * Programmer: Gerd Herber
```

Comment:
Gerd Heber

Reply:
I just removed the plus signs - didn't check name spelling.  :)